subworkflows/local/prepare_genome.nf

```diff
@@ -148,7 +148,7 @@ workflow PREPARE_GENOME {
     if (prepare_tool_index == 'bowtie2') {
         if (params.bowtie2_index) {
             if (params.bowtie2_index.endsWith('.tar.gz')) {
-                ch_bowtie2_index = UNTAR_BOWTIE2_INDEX ( params.bowtie2_index ).untar
+                ch_bowtie2_index = UNTAR_BOWTIE2_INDEX ( [], params.bowtie2_index ).untar.map{it -> it[1]}
```

Comment:
```suggestion
                ch_bowtie2_index = UNTAR_BOWTIE2_INDEX ( [], params.bowtie2_index ).untar.map{ it[1] }
```

Reply:
Addressed in [33d3bdd](https://github.com/nf-core/chipseq/pull/271/commits/33d3bdda056cc66bd6c9070bc0abf670e9c6c229)